packages/app-tos/vite.config.ts

```diff
@@ -49,6 +49,10 @@ export default defineConfig({
     rollupOptions: {
       external: [
         'react',
+        'next',
+        'zod',
+        'kysely',
+        'viem',
         '@safe-global/safe-apps-sdk',
         'jsonwebtoken',
         '@summerfi/serverless-shared',
```

Comment:
Not enough unfortunately, `vite` treats `next` and `next/server` (same with `viem`) as something else :(

```suggestion
      external: [
        'react',
        'next',
        'next/server',
        'zod',
        'kysely',
        'viem',
        'viem/chains',
        '@safe-global/safe-apps-sdk',
        'jsonwebtoken',
        '@summerfi/serverless-shared',
      ],
```